game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -2262,7 +2262,6 @@ private void showGame() {
               historySyncher.deactivate();
               historySyncher = null;
             }
-            historyPanel.goToEnd();
```

Comment:
I'm concerned this call might have some required unintended side-effect on the underlying GameData which might break things, but ok 🤷